### ISSUE TITLE
How to find pairs of modified methods?

### ISSUE THREAD

**Issue by oathaha** [microsoft/CodeBERT#231]
Given a source code file where some of its methods are modified, how do you get the pairs of modified methods?

Thank you.

**Comment by guoday** [comment]
Sorry, I don't know what you discuss. Can you give me details?

**Comment by oathaha** [comment]
I am sorry that I did not explain clearly. What I mean is like this.
Suppose we have A.java and this java file contains 4 methods (i.e., Method1(), Method2(), Method3(), Method4() )

Suppose Method2() and Method4() are modified, how did you get the pairs of old and new modified methods (Method2() and Method4() )

**Comment by guoday** [comment]
Do you ask about codereviewer?

**Comment by oathaha** [comment]
Yes. I would like to know how you prepare the triplets (original code, reviewer comment, revised code) to train CodeReviewer

**Comment by celbree** [comment]
This repo would be helpful for you. https://github.com/Lizhmq/CodeReviewProcesser, which is our data-processing code for CodeReviewer.

**Comment by oathaha** [comment]
Thank you very much.
